index.html

```diff
@@ -119,7 +129,7 @@ <h2 class="title">Co-Founders</h2>
               
                 </div>
                   <div class="col-sm">
-                  <img src="assets/Pranav.jpeg" alt="Person" class="card__image">
+                  <img src="assets/Pranav2.jpeg" alt="Person" class="card__image">
```

Comment:
Maybe just keep one Pranav picture in assets?

Reply:
Done!